packages/bindings/src/darwin_list.cpp

```diff
@@ -321,20 +324,23 @@ void EIO_List(uv_work_t* req) {
   }
 }
 
-void EIO_AfterList(uv_work_t* req) {
-  Nan::HandleScope scope;
+void EIO_AfterList(napi_env env, napi_status status, void* req) {
+  Napi::HandleScope scope(env);
 
-  ListBaton* data = static_cast<ListBaton*>(req->data);
+  ListBaton* data = (ListBaton*)req;
 
-  v8::Local<v8::Value> argv[2];
+  std::vector<napi_value> args;
+  args.reserve(2);
   if (data->errorString[0]) {
-    argv[0] = v8::Exception::Error(Nan::New<v8::String>(data->errorString).ToLocalChecked());
-    argv[1] = Nan::Undefined();
+    args.push_back(Napi::String::New(env, data->errorString));
+    napi_value undefined;
+    status = napi_get_undefined(env, &undefined);

```

Comment:
another unchecked call to a `napi_*` method. If you construct a `Napi::Env`, you can use `Napi::Env().Undefined()`